dm/syncer/syncer.go

```diff
@@ -1853,6 +1854,27 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 		return nil
 	}
 
+	advanceLatestLocationGtidSet := func(e *replication.BinlogEvent) error {
+		if _, ok := e.Event.(*replication.MariadbGTIDEvent); ok {
+			gtidSet, err2 := gtid.ParserGTID(s.cfg.Flavor, currentGTID)
```

Comment:
At L2279 we get gtidSet as string, here we parse gtid string to gtidSet, can we improve it?

Reply:
https://github.com/go-mysql-org/go-mysql/blob/4b20384/replication/binlogsyncer.go#L787
GTIDSet only accepts updates by string now. We need to update go-mysql-org if we don't want to parse gtid string again. Maybe later.